admin/class-paper-presenter.php

```diff
@@ -59,7 +62,17 @@ public function __construct( $title, $view_file, array $settings = array() ) {
 	 * @return string The rendered paper.
 	 */
 	public function get_output() {
-		extract( $this->get_view_variables(), EXTR_SKIP );
+		$view_variables = $this->get_view_variables();
+
+		extract( $view_variables, EXTR_SKIP );
```

Comment:
Just a note that all the usages of `extract()` still [need to be audited and replaced](https://github.com/Yoast/wordpress-seo/blob/4951550c59d56db27e6c92a95d6b80dd9bc9f4fe/phpcs.xml.dist#L22).

Reply:
Yes, but this is not possible in this case because the API is set up in a way that any number of view variables can be passed to the `class-paper-presenter.php` and then be passed to the view. I was very much aware of this issue, but it's not something I want to address in this PR due to its current complexity and size.

Reply:
Sure, it was just a note 🙂